tests/validation/convergence_test.hpp

```diff
@@ -2,10 +2,11 @@
 
 #include <util/filter.hpp>
 #include <util/rangeutil.hpp>
+#include <cell.hpp>
 
 #include <json/json.hpp>
 
-#include "gtest.h"
+#include "../gtest.h"
```

Comment:
It should be either `<gtest.h>` or `"../gtest.h" in my opinion. Isn't using `"..."` for paths relative to this one, and `<...>` for paths set with `-I -isystem` flags consistent?

The pattern "../gtest.h" in all of the unit tests.